demo.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash
 
-#IP=$(docker-machine env dev-nfs && docker-machine ip)
-IP=127.0.0.1
+SSO_API="https://beta-sso.dina-web.net/auth/realms/dina/tokens/grants/access"
+DW_API="https://beta-api.dina-web.net/collections/v0"
 
-RESULT=$(curl --data "grant_type=password&client_id=dina-rest-endpoint&username=reporter&password=reporter" http://$IP:8080/auth/realms/dina/tokens/grants/access)
-#<http://192.168.99.100:8080/auth/realms/dina/tokens/grants/access%60>
+RESULT=$(curl -s --data "grant_type=password&client_id=dina-rest&username=reporter&password=markus" \
```

Comment:
No, not really. I'd like to have "reporter". But I couldn't easily change back the settings to use "reporter" as password, because of an error I wasn't able to easily fix, something like "keycloak changes are pending". It would be nice to have programmatic way of adding a client rather than making ui settings in keycloak and then dumping into a mysql blob that is re-applied to the db. We have that in the dw-classifications. But I guess some time needs to be spent looking at the Keycloak REST API to figure out how to automate the required steps...